### DISCUSSION TITLE
VPS Question

### DISCUSSION THREAD

**Discussion by chop249** [Ysurac/openmptcprouter#2424] [Q&A]
Can the VPS be run on a Windows machine running VMWare? I thought about using a Raspberry Pi but seeing as that won't work I'm looking at other options. I would prefer not to have another computer running if I don't have to.

**Comment by Ysurac** [comment]
Yes VPS part can be run on any Hypervisor (VMWare, KVM, Virtualbox,...)
But VPS must be at a distant location with a connection faster than aggregated connections.

**Reply by chop249** [reply]
Is there a tutorial anywhere on how to do it via VMWare?

**Reply by Ysurac** [reply]
You need to use or install a Debian 10 image and then follow the documentation.
